packages/terra-table/docs/TableSubsection.md

```diff
@@ -0,0 +1,33 @@
+# Terra Table Subsection
+
+The Terra Table Subsection is a sub component that creates a subsection header based on the given props and groups the child rows beneath it. The subsection header has the ability to toggle the display of its child rows given the isCollapsible and isCollapsed props. For a subsection to be collapsed both isCollapsible and isCollapsed. Each section handles its collapsible and collapsed state.
+
+## Getting Started
```

Comment:
Do we need the Getting Started sections in each sub-component's readme?

Reply:
Updated.